app.R

```diff
@@ -72,29 +97,31 @@ ui <- fluidPage(
               tabPanel("1 Single",
                       sidebarLayout(position = "left",
                                     sidebarPanel(
-                                      actionButton("resetSingleForm", "Clear form!"),
-                                      actionButton("randomSpecies", "Random species!"),
+                                      #actionButton("resetSingleForm", "Clear form!"),
+                                      #actionButton("randomSpecies", "Random species!"),
   
                                       fluidRow(
                                         column(8, align="center", offset = 2,
-                                               tags$h4("Enter species")
+                                               tags$h4("Enter a species"),
```

Comment:
This looks better, except I'd maybe reducing the heading to just "Enter a species" and change the button to read "Random" or descriptive like that (as per Tarciso's suggestions).

Reply:
thanks, now working with Random Species - I think there is enough space, even for long species names

Reply:
For the headings - yes, I think good, but it was getting a bit cluttered. and you do have a kind of heading with the hide/show buttons. Maybe wait for some more feedback.